user/troubleshooting/vpn-troubleshooting.md

```diff
@@ -31,3 +31,20 @@ After suspend/resume, OpenVPN may not automatically reconnect. In order to get i
 After setting up OpenVPN and restarting the VM, you may be repeatedly getting the popup "Ready to start link", but the VPN isn't connected.
 
 To figure out the root of the problem, check the VPN logs in `/var/logs/syslog`. The log may reveal issues like missing OpenVPN libraries, which you can then install.
+
+## `notify-send` induced failure
+[Some VPN guides](https://github.com/Qubes-Community/Contents/blob/master/docs/configuration/vpn.md) use complex scripts that by themselves include call of `notify-send`, yet some images may not contain this tool or may not have it working properly.
+For instance calling `notify-send` on `fedora-36` template VM gives:
+```
+Failed to execute child process “dbus-launch” (No such file or directory)
+```
+
+To check this tool is working properly run:
+```bash
+sudo notify-send "$(hostname): Test notify-send OK" --icon=network-idle
+```
```

Comment:
Under which circumstances should this be working? root user shouldn't have a working dbus session, so it's unlikely notify-send would ever work here.

Reply:
see https://github.com/QubesOS/qubes-issues/issues/8289